src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -564,6 +619,29 @@ div.sf-toolbar  .sf-toolbar-block .sf-toolbar-info-piece.sf-toolbar-info-php-ext
     font-size: 12px;
 }
 
+/***** Toolbar Redirection Intercept *****/
```

Comment:
As the redirection interception is a dedicated page rendered by Symfony, I would not put those styles in the toolbar CSS. This page is free to have additional CSS that is not restricted by scoping needs.